test/functional/normal/fold_spec.lua

```diff
@@ -44,4 +48,113 @@ describe('Folds', function()
     8
     9]])
   end)
+  it('updates correctly on :read', function()
+    -- luacheck: ignore 621
+    helpers.write_file('./to_read.txt', [[
```

Comment:
generally we use `Xtest-foo` to name test files. ugly, but better to stay consistent. (If someone wants to change the convention later, it will be easier to find all cases)